src/LED_strip/led_strip.cpp

```diff
@@ -537,7 +537,9 @@ String getFieldValue(const char * name)
   Field field = getField(name);
   if (field.getValue)
   {
-    DynamicJsonDocument doc(1024);
+    // Buffer size: Mode names (164 modes * 20 chars = ~3280 bytes) or palette names (48 * 20 = ~960 bytes)
+    // Use larger size to handle worst case: 3584 bytes
+    DynamicJsonDocument doc(3584);
```

Comment:
The function returns the value of a single field as string.  all values are cast to uint32 and we return this as string. So please adjust the buffer to consider an uint32_t + overhead.